src/libnvme-mi.map

```diff
@@ -61,6 +61,8 @@ LIBNVME_MI_1_1 {
 		nvme_mi_open_mctp;
 		nvme_mi_scan_mctp;
 		nvme_mi_scan_ep;
+		nvme_mi_service_primitive_control;
+
```

Comment:
minor: no need for this blank line

Reply:
And it should go to a new section, see the libnvme.map file as example.

Reply:
done, put it in the 1_10